src/utils/ProtocolMetrics.ts

```diff
@@ -216,21 +290,25 @@ export function updateProtocolMetrics(transaction: Transaction): void {
     pm.sKlimaCirculatingSupply = getSklimaSupply(transaction)
 
     //KLIMA Price
-    pm.klimaPrice = getKLIMAUSDRate()
+    //pm.klimaPrice = getKLIMAUSDRate()
```

Comment:
Was this commented out deliberately?

Reply:
It was. Currently there is a divide by 0 error when attempting to index from the starting block defined in subgraph.yaml. Was going add the price function updates as their own commit and un comment at that time.